integration/common-provision.sh

```diff
@@ -33,14 +33,17 @@ openssl req -x509 -newkey rsa:2048 -keyout $CERTPATH/key.pem -out $CERTPATH/cert
 
 go version
 
+# check.go assumes rake to be present (it invokes `rake install`)
+# gcc is required to `go install` certain packages, probably those that `import "C"`
+# (p2/bin/p2-exec, opencontainers/runc/libcontainer/system, mattn/go-sqlite3)
+# git is required to `go get`
+sudo yum install -y rubygem-rake gcc git
```

Comment:
moved up, because as I say, we need `gcc` to `go install`, and there is a `go install` below.